packages/cli/src/ceramic-daemon.ts

```diff
@@ -927,6 +932,7 @@ export class CeramicDaemon {
   }
 
   async startIndexingModels(req: Request, res: Response): Promise<void> {
+    res.header('Deprecation', MODELS_DEPRECATION_AT)
     await this._processAdminModelsMutationRequest(req, res, {
       type: 'modelIDs',
       method: (modelIDs) => this.ceramic.admin.startIndexingModels(modelIDs),
```

Comment:
nitpick, but it seems odd to rewrite `getIndexedModels` in terms of `getIndexedModelData` but leave this and `stopIndexingModels` using dedicated methods on the underlying admin api.  I'd either leave `getIndexedModels` alone or change all 3, just so we're consistent

It's just code structure though so doesn't really matter, can always be changed later

Reply:
The underlying calls do the forwarding, so I'll change these back for consistency.